custom/languages/Dutch/admin.php

```diff
@@ -16,7 +16,7 @@
 	 *  Admin controle paneel
 	 */
 	// Login
-	're-authenticate' => 'Gelieve opnieuw te verifiëren',
+	're-authenticate' => 'Gelieve opnieuw te verifiÃ«ren',
```

Comment:
I cloned it with ssh and edited with winscp. It's something with my characterset.